packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.module.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@uirouter/angularjs';
+import '@ovh-ux/manager-core';
+
+import agoraOrder from '../../order/agora';
+import legacyOrder from '../../order/legacy';
+import routing from './order.routing';
+
+const moduleName = 'ovhManagerPciProjectFailoverIpsOnboardingOrder';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'pascalprecht.translate',
+    'ovhManagerCore',
+    agoraOrder,
```

Comment:
Avoid using `agora` in the name, you can call it `order` :)

Reply:
Will be handle in an upcoming refactor:)